src/components/item.jsx

```diff
@@ -52,11 +69,11 @@ class Item extends React.Component {
       <div className="item-view">
         <div key={this.props.itemId} className="item">
           ItemId: {this.props.itemId} <br/>
-          {/* Name: {this.state.itemName} <br/>
-          Systems: {this.state.availability} <br/>
-          Good Reviews: {this.state.feedback.positiveReviews}% <br/>
-          Total Reviews: {this.state.feedback.numberOfReviews} <br/>
-          Reviews From: {this.state.feedback.reviewsFrom} <br/> */}
+          {/* Name: {this.state.itemName} <br/> */}
+          {/* Systems: {this.state.availability} <br/> */}
+          {/* Good Reviews: {this.state.feedback.positiveReviews}% <br/> */}
+          {/* Total Reviews: {this.state.feedback.numberOfReviews} <br/> */}
+          {/* Reviews From: {this.state.feedback.reviewsFrom} <br/> */}
```

Comment:
I've never seen a change like this before. Does this way allow you to toggle line by line as opposed to the deleted being for the whole block?

Reply:
Exactly.